library/src/main/java/com/getstream/sdk/chat/model/ModelType.kt

```diff
@@ -48,4 +49,18 @@ object ModelType {
     const val action_send = "send"
     const val action_shuffle = "shuffle"
     const val action_cancel = "cancel"
+
+    val supportedMimeTypes = listOf(
```

Comment:
I see it duplicates `StorageHelper::supportedFilesMimeTypes`. I'd like to keep only one

Reply:
Agree, I've removed `StorageHelper::supportedFilesMimeTypes` in #677 (I didn't want to make too many changes in this PR)